Formula/libxml2.rb

```diff
@@ -41,7 +42,7 @@ class Libxml2 < Formula
 
   keg_only :provided_by_macos
 
-  depends_on "python@3.9" => [:build, :test]
+  depends_on "python@3.10" => [:build, :test]
```

Comment:
If we want to change this, we need to check that the following formulae don't make use of `libxml2`'s Python bindings:
```
anjuta
at-spi2-atk
gdal
gtk-doc
gupnp
itstool
yelp-tools
```

Reply:
`anjuta` makes use of those bindings. 😕

Reply:
Migrating `anjuta` will be a whole thing (it has `glib` in its dep tree, and that connects it to a pretty large fragment of homebrew-core dependency graph), so I suggest we stick with `python@3.9` for now.

When you make changes here can you squash your commits together? Thanks!